app/server/services/s2s.ts

```diff
@@ -0,0 +1,108 @@
+const config = require('config');
```

Comment:
This file is just a copy and paste from the tests. If we are thinking of refactoring this might be worth trying to reuse it in the tests